code/game/machinery/decontamination.dm

```diff
@@ -1,7 +1,7 @@
 // DECONTAMINATION STORAGE UNIT!! /////////////////
 /obj/machinery/decontamination_unit
 	name = "advanced decontamination storage unit"
-	desc = "This is a more advanced version of the industrial suit storage unit developed by the NT science and engineering team. It is capable of removing organic radiation as well as contaminated equipment. Can store upto 1000."
+	desc = "This is a more advanced version of the industrial suit storage unit developed by the NT science and engineering team. It is capable of removing organic radiation as well as contaminated equipment. Can store up to 1000 items."
```

Comment:
Remove the up to 1000 items part entirely since it's already handled in the extra examine text based on parts (which updates with the actual storage amount)